src/bootstrap.php

```diff
@@ -1812,6 +1829,89 @@ function (Accept $type, string $id) use ($app) {
     'application/vnd.elife.profile+json; version=1'
 ));
 
+$app->get('/promotional-collections', function (Request $request, Accept $type) use ($app) {
+    $promotionalCollections = $app['promotional-collections'];
+
+    $page = $request->query->get('page', 1);
+    $perPage = $request->query->get('per-page', 10);
+    $subjects = (array) $request->query->get('subject', []);
+    $containing = (array) $request->query->get('containing', []);
+
+    if (false === empty($subjects)) {
+        $promotionalCollections = array_filter($promotionalCollections, function ($promotionalCollection) use ($subjects) {
+            $promotionalCollectionsSubjects = array_map(function (array $subject) {
+                return $subject['id'];
+            }, $promotionalCollection['subjects'] ?? []);
+
+            return count(array_intersect($subjects, $promotionalCollectionsSubjects));
+        });
+    }
+
+    if (false === empty($containing)) {
+        $promotionalCollections = array_filter($promotionalCollections, function ($promotionalCollection) use ($containing) {
+            foreach ($promotionalCollection['content'] as $item) {
+                if (in_array("{$item['type']}/{$item['id']}", $containing)) {
+                    return true;
+                }
+            }
+
+            return false;
+        });
+    }
+
+    $content = [
+        'total' => count($promotionalCollections),
+        'items' => [],
+    ];
+
+    if ('asc' === $request->query->get('order', 'desc')) {
+        $promotionalCollections = array_reverse($promotionalCollections);
+    }
+
+    $promotionalCollections = array_slice($promotionalCollections, ($page * $perPage) - $perPage, $perPage);
+
+    if (0 === count($promotionalCollections) && $page > 1) {
+        throw new NotFoundHttpException('No page '.$page);
+    }
+
+    foreach ($promotionalCollections as $i => $promotionalCollection) {
+        unset($promotionalCollection['editors']);
```

Comment:
the most relevant bit with the properties that are different from the `collection` ones